homework/index.js

```diff
@@ -29,26 +41,66 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function renderRepoDetails(repo, innerul) {
+      let li;
+      // insert repo name in a li which includes a label tag and anchor tag
+      li = createAndAppend('li', innerul);
+      createAndAppend('label', li, { text: `Repository : `, class: "label"});
+      createAndAppend('a', li, { text: `${repo.name}  `, class: "info", href:"#"});  
+    
+  
+      // insert repo description 
+      li = createAndAppend('li', innerul);
+      createAndAppend('label', li, { text: `Description : `, class: "label"});
+      (repo.description === null) ? createAndAppend('p', li, { text: `${repo.name}  `, class: "info"}) : createAndAppend('p', li, { text: `${repo.description}  `, class: "info"});
```

Comment:
Cool you've made it a lot more compact! Awesome that you've improved it 😄 

One common pitfall with compact code can be that readability is impacted.
It's always a balance between brevity and readability. I'll give you my take on this line

```suggestion
      let descText = (repo.description === null ? repo.name : repo.description);
      createAndAppend('p', li, { text: `${descText}  `, class: "info"});
```

Reply:
Great, I really like your suggestion.